tests/core/basic20/task.yaml

```
@@ -58,4 +61,11 @@ execute: |
     done
 
     # ensure the "snap recovery" command works
-    snap recovery | MATCH '[0-9]+\ +canonical\*\ +ubuntu-core-20-[^ ]+\ +current'
+    MODEL="$(snap model | grep '^model' | awk '{ print $2 }')"
+    BRAND="$(snap model | grep '^brand' | cut -d' ' -f2- | xargs)"
```

I think given the fact that `snap recovery` uses the `shortPublisher` helper when outputting the brand, and doesn't do any handling of display-name for the model, we should actually use `--verbose` for both of these

```suggestion
    MODEL="$(snap model --verbose | grep '^model' | awk '{ print $2 }')"
    BRAND="$(snap model --verbose | grep '^brand' | cut -d' ' -f2- | xargs)"
```

---

done